app/controllers/admin/enterprises_controller.rb

```diff
@@ -44,7 +45,11 @@ def index
     def edit
       @object = Enterprise.where(permalink: params[:id]).
         includes(users: [:ship_address, :bill_address]).first
-      super
+      if params[:stimulus]
```

Comment:
I'm new to Stimulus. Is this an accepted convention to render a CableCar response? It looks a bit at odds with Rails conventions...

Reply:
Yes, it's something that comes built in with Stimulus Reflex but not with StimulusJs
```
When StimulusReflex is rendering your template, an instance variable named @stimulus_reflex is available to your Rails controller and set to true.

You can use this flag to create branching logic to control how the template might look different if it's a Reflex vs normal page refresh.
```